compiler/rustc_hir_analysis/src/check/check.rs

```diff
@@ -805,7 +805,19 @@ fn check_impl_items_against_trait<'tcx>(
                 .is_some_and(|node_item| node_item.item.defaultness(tcx).has_value());
 
             if !is_implemented && tcx.defaultness(impl_id).is_final() {
-                missing_items.push(tcx.associated_item(trait_item_id));
+                let infcx = tcx.infer_ctxt().build();
```

Comment:
This probably should use intercrate mode - https://github.com/rust-lang/rust/issues/20021#issuecomment-242547315

Reply:
That's necessary, but not sufficient. If there are no impls at all for a foreign trait, we'll still error. If we wanted to do this, it could get ugly, because we'd need to invoke some of coherence ourselves or sth (making sure either the type or the trait is local).